metagpt/ext/ai_writer/app_demo.py

```diff
@@ -0,0 +1,364 @@
+import gradio as gr
+import json,os ,shutil
```

Comment:
You need to use pre-commit. Or you can use the built-in function of vscode / pycharm to reformat. However, the submitted code needs to comply with most of the PEP8 standards, otherwise it will be difficult for us to accept it.

Reply:
I have been modified, please check again 
thanks

Reply:
我查看了文件，根据以上 CR 意见，这些文件还需要进一步的修改

Reply:
好的，我已对README文件和.pre-commit-config.yaml 文件进行修订